lib/ProductOpener/Display.pm

```diff
@@ -328,13 +352,22 @@ sub analyze_request($)
 {
 	my $request_ref = shift;
 	
+	print STDERR "analyze_request : query_string 0 : $request_ref->{query_string} \n";
+	
+	
 	# http://world.openfoodfacts.org/?utm_content=bufferbd4aa&utm_medium=social&utm_source=twitter.com&utm_campaign=buffer
 	# http://world.openfoodfacts.org/?ref=producthunt
 	
 	if ($request_ref->{query_string} =~ /(\&|\?)(utm_|ref=)/) {
 		$request_ref->{query_string} = $`;
-	}	
+	}
 	
+	# cc and lc query overrides have already been consumed by init()
+	# FIXME / TODO: just remove those 2 parameters, leave others.
+	if ($request_ref->{query_string} =~ /(\&|\?)(cc|lc)/) {
+		$request_ref->{query_string} = $`;
```

Comment:
Couldn't they just be kept in so that other parts of the server can detect the override as well?